include/kernel/page-alloc.h

```diff
@@ -6,6 +6,27 @@
 #define PAGE_SIZE_SMALL (1ULL << 12)
 #define PAGE_SIZE_LARGE (1ULL << 21)
 
+
+/// \file       include:kernel:page-alloc.h
+/// \brief      File constitutes page memory allocation and deallocation methods
+///
+/// \fn         page_alloc_init(void)
+/// \brief      page_alloc_init initialzes the page allocation
```

Comment:
Can you please move these comments to the individual functions that they document, and use the regular format?

Reply:
The `.h` files say, header files can have Doxygen Docs present, it would like having per method when the docs would be generated. See: http://www.doxygen.nl/manual/commands.html.